Documentation/guides/OneDotNet.md

```diff
@@ -135,6 +135,38 @@ Default Android related file globbing behavior is defined in `Microsoft.Android.
 This behavior can be disabled for Android items by setting `$(EnableDefaultAndroidItems)` to `false`, or
 all default item inclusion behavior can be disabled by setting `$(EnableDefaultItems)` to `false`.
 
+## Linker (ILLink)
+
+.NET 5 and higher has new [settings for the linker][linker]:
+
+* `<PublishTrimmed>true</PublishTrimmed>`
+* `<TrimMode>copyused</TrimMode>` - Enable assembly-level trimming.
+* `<TrimMode>link</TrimMode>` - Enable member-level trimming.
+
+In Android application projects by default, `Debug` builds will not
+use the linker and `Release` builds will set `PublishTrimmed=true` and
+`TrimMode=link`. `TrimMode=copyused` is the default the dotnet/sdk,
+but it doesn't not seem to be appropriate for mobile applications.
+Developers can still opt into `TrimMode=copyused` if needed.
+
+If the legacy `AndroidLinkMode` setting is used, both `SdkOnly` and
+`Full` will default to equivalent linker settings:
+
+* `<PublishTrimmed>true</PublishTrimmed>`
+* `<TrimMode>link</TrimMode>`
+
+With `AndroidLinkMode=SdkOnly` only BCL and SDK assemblies marked with
+`%(Trimmable)` will be linked at the member level.
+`AndroidLinkMode=Full` will set `%(TrimMode)=link` on all .NET
+assemblies similar to the example in the [trimming
+documentation][linker-full].
+
+It is recommended to migrate to the new linker settings, as
+`AndroidLinkMode` will eventually be deprecated.
```

Comment:
If this will eventually be deprecated, we need to start emitting a deprecation warning around its use.

Which just raises a different question: is there/can there be a mechanism that is usable both in the Current World Order *and* in .NET 6?  Or *is* `$(AndroidLinkMode)` that mechanism, and you're suggesting that support for `$(AndroidLinkMode)` need not be removed until "later" (.NET 7? 8?).